### PR TITLE
Prep for Release 0.13.0

### PR DESCRIPTION
**Pull request by neelip**
- package.json and package-lock.json updates (version and name)
- updates to CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.13.0 - 2020-05-28
+### Added
+- PostgreSQL support
+  - Basic CRUD support added to PostgreSQL databases and integrated with the exisitng extension; updates to Tables is not yet supported
```
Comment (wwlorey) [inline CHANGELOG.md:8]: What about this: "Create and view PostgreSQL servers, databases, and functions"

I'm not a huge fan of using "CRUD". I don't know if this is just me, but I didn't know what it meant for a long time 😂 And we probably don't need to list *all* of the possible actions you can take on a resource

Comment (nturinski) [reply CHANGELOG.md:8]: Whenever I hear CRUD, I think 💩

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+  - Functions and stored procedures CRUD support
+### Changed
+- Top level naming convention is updated from "CosmosDB" to "Azure Databases"
```
[the remark this answers is not in the document]
Comment (nturinski) [reply CHANGELOG.md:11]: @fiveisprime Could you weigh in on this?  I personally feel like just adding a small note in the "Changed" section seems pretty unceremonious.  Should we put something on the top to explain the rebrand decision at all?  Like how we're expanding our database support beyond what's in Cosmos, so we are rebranding to include other databases, starting with Postgres?

Comment (EricJizbaMSFT) [inline CHANGELOG.md:11]: Don't forget to mention something like this:
```
- Viewing and querying Azure Cosmos DB graphs has been moved to a separate graph-specific extension
```

[the remark this answers is not in the document]
Comment (fiveisprime) [reply CHANGELOG.md:11]: Yeah, make it obvious that it's important.

"In this release, we're rebranding the Cosmos DB Extension to Azure Databases to expand our database support beyond just Cosmos DB. This release also marks the first new database engine: PostgreSQL with other database engines coming later this year."

How is that?

Comment (nturinski) [reply CHANGELOG.md:11]: Should we add a link to it?  It feels like the right thing to do, but I'm not sure we're exactly _trying_ to lead people to it?

Comment (EricJizbaMSFT) [reply CHANGELOG.md:11]: We handle this directly in-product when they click a graph. IMO that's the most important piece and I don't really have a preference for the docs/changelog

Comment (neelip) [reply CHANGELOG.md:11]: I will just add it, just for record keeping purposes

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,18 @@
...
+## 0.13.0 - 2020-05-28
+In this release, we're rebranding the Cosmos DB Extension to Azure Databases to expand our database support beyond just Cosmos DB. This release also marks the first new database engine: PostgreSQL with other database engines coming later this year.
+### Added
+- PostgreSQL support
+  - Create and view PostgreSQL servers, databases, and functions.
+  - Create, edit, and execute PostgreSQL queries
+### Fixed
+- [Bugs fixed](https://github.com/microsoft/vscode-cosmosdb/issues?q=is%3Aissue+milestone%3A0.13.0+is%3Aclosed)
+### Removed
+- Viewing and querying Azure Cosmos DB graphs has been moved to a separate graph-specific extension,
```
Comment (wwlorey) [inline CHANGELOG.md:14]: nit: Should this go under the "Changed" subheading instead?

Comment (neelip) [reply CHANGELOG.md:14]: That makes sense, given its not completely removed

**Pull request merged**